diff --git a/dist/index.mjs b/dist/index.mjs
index a2fa5cad35283ab84691093f69db88640111c5ea..44621a0958e76503f333c1340b3a697001cea6c3 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -9679,16 +9679,12 @@ const Nav = ({
         className: `group w-full px-6 py-3 gap-2 flex justify-between items-center transition-colors duration-150 ease-out ${open2 ? `bg-gray-50` : `bg-transparent`}`
       },
       /* @__PURE__ */ React.createElement("span", { className: "text-left inline-flex items-center text-xl tracking-wide text-gray-800 flex-1 gap-1 opacity-80 group-hover:opacity-100 transition-opacity duration-150 ease-out" }, /* @__PURE__ */ React.createElement(
-        "svg",
+        "img",
         {
-          viewBox: "0 0 32 32",
-          fill: "#EC4815",
-          xmlns: "http://www.w3.org/2000/svg",
+          src: "/fiesta.svg",
           className: "w-10 h-auto -ml-1"
-        },
-        /* @__PURE__ */ React.createElement("path", { d: "M18.6466 14.5553C19.9018 13.5141 20.458 7.36086 21.0014 5.14903C21.5447 2.9372 23.7919 3.04938 23.7919 3.04938C23.7919 3.04938 23.2085 4.06764 23.4464 4.82751C23.6844 5.58738 25.3145 6.26662 25.3145 6.26662L24.9629 7.19622C24.9629 7.19622 24.2288 7.10204 23.7919 7.9785C23.355 8.85496 24.3392 17.4442 24.3392 17.4442C24.3392 17.4442 21.4469 22.7275 21.4469 24.9206C21.4469 27.1136 22.4819 28.9515 22.4819 28.9515H21.0296C21.0296 28.9515 18.899 26.4086 18.462 25.1378C18.0251 23.8669 18.1998 22.596 18.1998 22.596C18.1998 22.596 15.8839 22.4646 13.8303 22.596C11.7767 22.7275 10.4072 24.498 10.16 25.4884C9.91287 26.4787 9.81048 28.9515 9.81048 28.9515H8.66211C7.96315 26.7882 7.40803 26.0129 7.70918 24.9206C8.54334 21.8949 8.37949 20.1788 8.18635 19.4145C7.99321 18.6501 6.68552 17.983 6.68552 17.983C7.32609 16.6741 7.97996 16.0452 10.7926 15.9796C13.6052 15.914 17.3915 15.5965 18.6466 14.5553Z" }),
-        /* @__PURE__ */ React.createElement("path", { d: "M11.1268 24.7939C11.1268 24.7939 11.4236 27.5481 13.0001 28.9516H14.3511C13.0001 27.4166 12.8527 23.4155 12.8527 23.4155C12.1656 23.6399 11.3045 24.3846 11.1268 24.7939Z" })
-      ), /* @__PURE__ */ React.createElement("span", null, "Tina")),
+          }
+      ), /* @__PURE__ */ React.createElement("span", null, "FIESTA")),
       /* @__PURE__ */ React.createElement(SyncErrorWidget, { cms }),
       /* @__PURE__ */ React.createElement(
         FiMoreVertical,
@@ -29773,8 +29769,8 @@ class ErrorBoundary extends React__default.Component {
               boxShadow: "0 6px 24px rgb(0 37 91 / 5%), 0 2px 4px rgb(0 37 91 / 3%)"
             }
           },
-          /* @__PURE__ */ React__default.createElement("h3", { style: { color: "#eb6337" } }, "TinaCMS Render Error"),
-          /* @__PURE__ */ React__default.createElement("p", null, "Tina caught an error while updating the page:"),
+          /* @__PURE__ */ React__default.createElement("h3", { style: { color: "#eb6337" } }, "FIESTA Render Error"),
+          /* @__PURE__ */ React__default.createElement("p", null, "FIESTA caught an error while updating the page:"),
           /* @__PURE__ */ React__default.createElement("pre", { style: { marginTop: "1rem", overflowX: "auto" } }, this.state.message),
           /* @__PURE__ */ React__default.createElement("br", null),
           /* @__PURE__ */ React__default.createElement("p", null, `If you've just updated the form, undo your most recent changes and click "refresh". If after a few refreshes, you're still encountering this error. There is a bigger issue with the site. Please reach out to your site admin.`),
@@ -29970,7 +29966,7 @@ const Loader = (props) => {
             fontWeight: "normal"
           }
         },
-        "Please wait, Tina is loading data..."
+        "Please wait, FIESTA is loading data..."
       )
     )
   ), props.children);
@@ -30265,7 +30261,7 @@ const PageBodyNarrow = ({ children }) => /* @__PURE__ */ React__default.createEl
 const DashboardPage = () => {
   return /* @__PURE__ */ React__default.createElement(GetCMS, null, (cms) => {
     var _a, _b;
-    return /* @__PURE__ */ React__default.createElement(PageWrapper, null, /* @__PURE__ */ React__default.createElement(React__default.Fragment, null, /* @__PURE__ */ React__default.createElement(PageHeader, { isLocalMode: (_b = (_a = cms.api) == null ? void 0 : _a.tina) == null ? void 0 : _b.isLocalMode }, /* @__PURE__ */ React__default.createElement("h3", { className: "text-2xl font-sans text-gray-700" }, "Welcome to Tina!")), /* @__PURE__ */ React__default.createElement(PageBodyNarrow, null, "This is your dashboard for editing or creating content. Select a collection on the left to begin.")));
+    return /* @__PURE__ */ React__default.createElement(PageWrapper, null, /* @__PURE__ */ React__default.createElement(React__default.Fragment, null, /* @__PURE__ */ React__default.createElement(PageHeader, { isLocalMode: (_b = (_a = cms.api) == null ? void 0 : _a.tina) == null ? void 0 : _b.isLocalMode }, /* @__PURE__ */ React__default.createElement("h3", { className: "text-2xl font-sans text-gray-700" }, "Welcome to FIESTA!")), /* @__PURE__ */ React__default.createElement(PageBodyNarrow, null, "This is your dashboard for editing or creating content. Select a collection on the left to begin.")));
   });
 };
 function RiHome2Line(props) {
@@ -30376,7 +30372,7 @@ const LoadingPage = () => /* @__PURE__ */ React__default.createElement(React__de
           fontWeight: "normal"
         }
       },
-      "Please wait, Tina is loading data..."
+      "Please wait, FIESTA is loading data..."
     )
   )
 ));
@@ -30962,25 +30958,8 @@ const CollectionListPage = () => {
               }
             }
           }
-        ))), /* @__PURE__ */ React__default.createElement("div", { className: "flex flex-1 flex-col gap-2 items-start w-full" }, searchEnabled ? /* @__PURE__ */ React__default.createElement(
-          SearchInput,
-          {
-            loading: _loading,
-            search,
-            setSearch,
-            searchInput,
-            setSearchInput
-          }
-        ) : /* @__PURE__ */ React__default.createElement(React__default.Fragment, null, /* @__PURE__ */ React__default.createElement("label", { className: "block font-sans text-xs font-semibold text-gray-500 whitespace-normal" }, "Search"), /* @__PURE__ */ React__default.createElement(
-          Message,
-          {
-            link: "https://tina.io/docs/reference/search/overview",
-            linkLabel: "Read The Docs",
-            type: "info",
-            size: "small"
-          },
-          "Search not configured."
-        )))), /* @__PURE__ */ React__default.createElement("div", { className: "flex flex-col md:flex-row items-start md:items-end gap-2 md:gap-0 pt-4 lg:pt-0" }, !collection.templates && allowCreate && /* @__PURE__ */ React__default.createElement(React__default.Fragment, null, allowCreateNestedFolder && /* @__PURE__ */ React__default.createElement(
+        ))),
+        ), /* @__PURE__ */ React__default.createElement("div", { className: "flex flex-col md:flex-row items-start md:items-end gap-2 md:gap-0 pt-4 lg:pt-0" }, !collection.templates && allowCreate && /* @__PURE__ */ React__default.createElement(React__default.Fragment, null, allowCreateNestedFolder && /* @__PURE__ */ React__default.createElement(
           Link,
           {
             onMouseDown: (evt) => {
@@ -31379,7 +31358,7 @@ const RenameModal = ({
   newRelativePath,
   setNewRelativePath
 }) => {
-  return /* @__PURE__ */ React__default.createElement(Modal, null, /* @__PURE__ */ React__default.createElement(PopupModal, null, /* @__PURE__ */ React__default.createElement(ModalHeader, { close: close2 }, "Rename ", filename), /* @__PURE__ */ React__default.createElement(ModalBody, { padded: true }, /* @__PURE__ */ React__default.createElement(React__default.Fragment, null, /* @__PURE__ */ React__default.createElement("p", { className: "mb-4" }, "Are you sure you want to rename ", /* @__PURE__ */ React__default.createElement("strong", null, filename), "? TinaCMS uses the filename as the ID; renaming this file could result in unresolved references."), /* @__PURE__ */ React__default.createElement(
+  return /* @__PURE__ */ React__default.createElement(Modal, null, /* @__PURE__ */ React__default.createElement(PopupModal, null, /* @__PURE__ */ React__default.createElement(ModalHeader, { close: close2 }, "Rename ", filename), /* @__PURE__ */ React__default.createElement(ModalBody, { padded: true }, /* @__PURE__ */ React__default.createElement(React__default.Fragment, null, /* @__PURE__ */ React__default.createElement("p", { className: "mb-4" }, "Are you sure you want to rename ", /* @__PURE__ */ React__default.createElement("strong", null, filename), "? FIESTA uses the filename as the ID; renaming this file could result in unresolved references."), /* @__PURE__ */ React__default.createElement(
     BaseTextField,
     {
       placeholder: "Enter a new name for the document's file",
@@ -31666,7 +31645,7 @@ const RenderForm$1 = ({
             cms.alerts.error(
               () => ErrorDialog({
                 title: defaultErrorText,
-                message: "Tina caught an error while creating the page",
+                message: "FIESTA caught an error while creating the page",
                 error
               })
             );
@@ -31893,7 +31872,7 @@ const RenderForm = ({
           cms.alerts.error(
             () => ErrorDialog({
               title: "There was a problem saving your document",
-              message: "Tina caught an error while updating the page",
+              message: "FIESTA caught an error while updating the page",
               error
             })
           );
@@ -32016,7 +31995,7 @@ const IndexingPage = () => {
         } catch {
           cms.alerts.error("Branch indexing failed.");
           setErrorMessage(
-            'Branch indexing failed, please check the Tina Cloud dashboard for more information. To try again chick "re-index" on the branch in the dashboard.'
+            'Branch indexing failed, please check the FIESTA dashboard for more information. To try again chick "re-index" on the branch in the dashboard.'
           );
           setState("error");
         }
@@ -32063,7 +32042,7 @@ const IndexingPage = () => {
         const foo = await tinaApi.createPullRequest({
           baseBranch,
           branch,
-          title: `${branch.replace("tina/", "").replace("-", " ")} (PR from TinaCMS)`
+          title: `${branch.replace("tina/", "").replace("-", " ")} (PR from FIESTA)`
         });
         console.log("PR created", foo);
         cms.alerts.success("Pull request created.");
@@ -32175,7 +32154,7 @@ const CheckSchema = ({
           );
         }
       }).catch((e) => {
-        if (e.message.includes("has not been indexed by Tina Cloud")) {
+        if (e.message.includes("has not been indexed by FIESTA")) {
           setSchemaMissingError(true);
         } else {
           cms.alerts.error(`Unexpected error checking schema: ${e}`);
